change/@fluentui-web-components-967265b4-5017-4bbb-b7cf-d88586cda5a9.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32350&commitId=1ab140487b5632510706206482ea626eb358da75&env=prod&clientType=FLUENTUI)


  <details><summary><b>Slider</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Default.chromium.png|48| Changed|
|Slider.Slider Steps Vertical.chromium.png|43| Changed|
|Slider.Default Dark Mode.chromium.png|38| Changed|
|Slider.Disabled Dark Mode.chromium.png|125| Changed|
|Slider.Size Medium.chromium.png|48| Changed|
|Slider.Default RTL.chromium.png|48| Changed|
|Slider.Size Small.chromium.png|19| Changed|
|Slider.Disabled.chromium.png|122| Changed|
|Slider.Vertical.chromium.png|49| Changed|
|Slider.Slider Steps.chromium.png|48| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>